### PR TITLE
PR for jRuby in the conda-forge channel 

### PR DESCRIPTION
**Pull request by abhi18av**
PR for Jruby to be included in `conda-forge`

### CODE REVIEW

**Comment by jakirkham** [comment]
cc @conda-forge/ruby

File: recipes/jruby/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  version: {{ version }}
+
+
+source:
+  url: https://s3.amazonaws.com/jruby.org/downloads/9.1.13.0/jruby-bin-9.1.13.0.tar.gz
+  sha256: 9d156646623ac2f27174721035b52572a4b05690db7c1293295aa2c04aad3908
+
+build:
+  number: 0
+  skip: true  # [win]
```
Comment (isuruf) [inline recipes/jruby/meta.yaml]: You need to have a build script to compile jruby. Otherwise this is just a package with no files.

Comment (abhi18av) [reply recipes/jruby/meta.yaml]: Ah, that's why the last line often say "Uploading no packages " ?

But for jruby / groovy etc we only really need to have a "brew install" like arrangement with the "openjdk" being a pre-requisite i.e. we only need to have the binary file within the Conda env.

This is quite similar to rust-feedstock or go-feedstock which don't build anything but place the proper binaries in the environment.

If this is not the right direction could you please help me out with this then ?

**Comment by abhi18av** [comment]
@isuruf mm, it seems I'm doing something wrong with the `bld.bat` and `build.sh`

**Pull request closed without merging**